src/patternfly/components/Table/table.scss

```diff
@@ -473,6 +473,14 @@
   justify-content: start;
   grid-template-columns: auto;
   grid-auto-columns: max-content;
+
+  .pf-m-nowrap & {
```

Comment:
can you scope these modifiers to the elements they apply to? If there were something outside of the table with `.pf-m-nowrap/truncate`, that would enable this CSS.